evalit/_base.py

```diff
@@ -123,35 +226,75 @@ def _sanity_check_config(self, cfg: Dict[str, str]) -> bool:
     def __classname__(self) -> str:
         return self.__class__.__name__
 
-    def __get_redacted_cfg(self) -> Dict[str, str]:
+    @staticmethod
+    def _get_redacted_cfg(cfg: Dict[str, str]) -> Dict[str, str]:
         """
         Returns the stored config in redacted form.
+
+        Args:
+            ```cfg```: ```Dict[str, str]```
+                Configuratin dictionary
+
+        Returns:
+            `Dict[str, str]` dictionary of same structure, but redacted
+
+        Example of dictionary structure:
+
+            .. code-block: python
+
+                {
+                    "source_token": "a****",
+                    "source_secret": "***s**rd",
+                    "source_s3_endpoint": "ht*p**/127.0**.*:80*0",
+                    "source_s3_bucket": "***",
+                    "source_s3_region": "u****s***",
+                    "dest_token": "*d*i*",
+                    "dest_secret": "p**s****",
+                    "dest_s3_endpoint": "htt*://127.0.*.1:*0**",
+                    "dest_s3_bucket": "**s*",
+                    "dest_s3_region": "u*-e*s***",
+                }
```

Comment:
Do we need to retain the length of the string? Wouldn't it be easier/better to use fixed length string and just have "first" or "last" values presented?